dlib/image_loader/image_loader.h

```diff
@@ -110,7 +110,11 @@ namespace dlib
             i += 4;
             a = buf[i]; b = buf[i+1]; c = buf[i+2]; d = buf[i+3];
             biHeight = a | (b<<8) | (c<<16) | (d<<24);
-
+            
+            bool topDown = biHeight < 0;
```

Comment:
Shouldn't this variable be called bottomUp?

Reply:
Yes, you are right. Fixed.